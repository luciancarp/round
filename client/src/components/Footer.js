import React, { Component } from 'react'
import { palette, spaces, fontSizes, device } from '../styles/styles'
import styled from 'styled-components'
import { FaFacebookF, FaTwitter, FaInstagram } from 'react-icons/fa'
import StyledLink from './common/StyledLink'
import { connect } from 'react-redux'
import PropTypes from 'prop-types'

import { logOutUser } from '../actions/authActions'

const StyledNavBar = styled.div`
  display: flex;
  margin: 0px;
  background-color: ${palette.darkBackgroundColor};
  height: 150px;
  padding: ${spaces.medium}px;
  flex-direction: row;
  align-items: center;
  justify-content: center;
`

const StyledNavBarContent = styled.div`
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;

  @media ${device.mobileS} {
    min-width: 100%;
  }

  @media ${device.laptop} {
    min-width: 1000px;
  }

  @media ${device.desktop} {
    min-width: 1400px;
  }
`

const StyledCopy = styled.span`
  font-size: ${fontSizes.regular}px;
  color: ${palette.darkTextSecundary};
`

const StyledLinks = styled.div`
  display: flex;
  flex-direction: row;
`

const StyledSocial = styled.div`
  margin-left: ${spaces.wide}px;
`

// const StyledLogo = styled.img`
//   height: 50px;
//   width: 50px;
// `

const StyledMap = styled.div`
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: center;
`

class Footer extends Component {
  onLogoutClick(e) {
    e.preventDefault()
    this.props.logOutUser()
  }

  render() {
    let currentYear = new Date().getFullYear()
    return (
      <div>
        <StyledNavBar>
          <StyledNavBarContent>
            {/* <Link to='/'>
              <StyledLogo src={require('../assets/images/logo_small.png')} />
            </Link> */}
            <StyledMap>
              <button onClick={e => this.onLogoutClick(e)}>Log Out</button>
              <StyledLink to='/login'>Login</StyledLink>
              <StyledLink to='/register'>Register</StyledLink>
              <StyledLink to='/profile'>Profile</StyledLink>
            </StyledMap>

            <StyledCopy>
              Â© Copyright {currentYear}.<br />
              All right reserved.
            </StyledCopy>
            <StyledLinks>
              <StyledSocial>
                <FaFacebookF size='30px' color={palette.darkTextSecundary} />
              </StyledSocial>
              <StyledSocial>
                <FaTwitter size='30px' color={palette.darkTextSecundary} />
              </StyledSocial>
              <StyledSocial>
                <FaInstagram size='30px' color={palette.darkTextSecundary} />
              </StyledSocial>
            </StyledLinks>
          </StyledNavBarContent>
        </StyledNavBar>
      </div>
    )
  }
}

Footer.propTypes = {
  auth: PropTypes.object.isRequired
}

const mapStateToProps = state => ({
  auth: state.auth
})

export default connect(
  mapStateToProps,
  { logOutUser }
)(Footer)
