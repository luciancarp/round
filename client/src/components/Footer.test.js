import React from 'react'
import { shallow } from 'enzyme'
import Footer from './Footer'

it('should render correctly with no props', () => {
  const component = shallow(<Footer />)
  expect(component).toMatchSnapshot()
})
