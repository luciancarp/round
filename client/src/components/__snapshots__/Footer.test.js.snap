// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly with no props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ForwardRef>
        <ForwardRef>
          <Link
            replace={false}
            to="/"
          >
            <ForwardRef
              src="logo_small.png"
            />
          </Link>
          <ForwardRef>
            © Copyright 
            2019
            .
            <br />
            All right reserved.
          </ForwardRef>
          <ForwardRef>
            <ForwardRef>
              <FaFacebookF
                color="#808080"
                size="30px"
              />
            </ForwardRef>
            <ForwardRef>
              <FaTwitter
                color="#808080"
                size="30px"
              />
            </ForwardRef>
            <ForwardRef>
              <FaInstagram
                color="#808080"
                size="30px"
              />
            </ForwardRef>
          </ForwardRef>
        </ForwardRef>
      </ForwardRef>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <ForwardRef>
          <Link
            replace={false}
            to="/"
          >
            <ForwardRef
              src="logo_small.png"
            />
          </Link>
          <ForwardRef>
            © Copyright 
            2019
            .
            <br />
            All right reserved.
          </ForwardRef>
          <ForwardRef>
            <ForwardRef>
              <FaFacebookF
                color="#808080"
                size="30px"
              />
            </ForwardRef>
            <ForwardRef>
              <FaTwitter
                color="#808080"
                size="30px"
              />
            </ForwardRef>
            <ForwardRef>
              <FaInstagram
                color="#808080"
                size="30px"
              />
            </ForwardRef>
          </ForwardRef>
        </ForwardRef>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <Link
              replace={false}
              to="/"
            >
              <ForwardRef
                src="logo_small.png"
              />
            </Link>,
            <ForwardRef>
              © Copyright 
              2019
              .
              <br />
              All right reserved.
            </ForwardRef>,
            <ForwardRef>
              <ForwardRef>
                <FaFacebookF
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
              <ForwardRef>
                <FaTwitter
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
              <ForwardRef>
                <FaInstagram
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
            </ForwardRef>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": <ForwardRef
                src="logo_small.png"
              />,
              "replace": false,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "src": "logo_small.png",
              },
              "ref": null,
              "rendered": null,
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-EHOje",
                  "isStatic": true,
                  "rules": Array [
                    "
  height: 50px;
  width: 50px;
",
                  ],
                },
                "displayName": "styled.img",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-EHOje",
                "target": "img",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                "© Copyright ",
                2019,
                ".",
                <br />,
                "All right reserved.",
              ],
            },
            "ref": null,
            "rendered": Array [
              "© Copyright ",
              2019,
              ".",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "br",
              },
              "All right reserved.",
            ],
            "type": Object {
              "$$typeof": Symbol(react.forward_ref),
              "attrs": Array [],
              "componentStyle": ComponentStyle {
                "componentId": "sc-htpNat",
                "isStatic": true,
                "rules": Array [
                  "
  font-size: ",
                  "14",
                  "px;
  color: ",
                  "#808080",
                  ";
",
                ],
              },
              "displayName": "styled.span",
              "foldedComponentIds": Array [],
              "render": [Function],
              "styledComponentId": "sc-htpNat",
              "target": "span",
              "toString": [Function],
              "warnTooManyClasses": [Function],
              "withComponent": [Function],
            },
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <ForwardRef>
                  <FaFacebookF
                    color="#808080"
                    size="30px"
                  />
                </ForwardRef>,
                <ForwardRef>
                  <FaTwitter
                    color="#808080"
                    size="30px"
                  />
                </ForwardRef>,
                <ForwardRef>
                  <FaInstagram
                    color="#808080"
                    size="30px"
                  />
                </ForwardRef>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <FaFacebookF
                    color="#808080"
                    size="30px"
                  />,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "color": "#808080",
                    "size": "30px",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "rules": Array [
                      "
  margin-left: ",
                      "22",
                      "px;
",
                    ],
                  },
                  "displayName": "styled.div",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "div",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <FaTwitter
                    color="#808080"
                    size="30px"
                  />,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "color": "#808080",
                    "size": "30px",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "rules": Array [
                      "
  margin-left: ",
                      "22",
                      "px;
",
                    ],
                  },
                  "displayName": "styled.div",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "div",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": <FaInstagram
                    color="#808080"
                    size="30px"
                  />,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "color": "#808080",
                    "size": "30px",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "rules": Array [
                      "
  margin-left: ",
                      "22",
                      "px;
",
                    ],
                  },
                  "displayName": "styled.div",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "div",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
            ],
            "type": Object {
              "$$typeof": Symbol(react.forward_ref),
              "attrs": Array [],
              "componentStyle": ComponentStyle {
                "componentId": "sc-bxivhb",
                "isStatic": true,
                "rules": Array [
                  "
  display: flex;
  flex-direction: row;
",
                ],
              },
              "displayName": "styled.div",
              "foldedComponentIds": Array [],
              "render": [Function],
              "styledComponentId": "sc-bxivhb",
              "target": "div",
              "toString": [Function],
              "warnTooManyClasses": [Function],
              "withComponent": [Function],
            },
          },
        ],
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bwzfXH",
            "isStatic": true,
            "rules": Array [
              "
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;

  @media ",
              "(min-width: 320px)",
              " {  
    min-width: 100%;
  }

    @media ",
              "(min-width: 1024px)",
              " {  
    min-width: 1000px;
  }

  @media ",
              "(min-width: 2560px)",
              " {
    min-width: 1400px;
  }
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bwzfXH",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  display: flex;
  margin: 0px;
  background-color: ",
            "#1f1f1f",
            ";
  height: 150px;
  padding: ",
            "14",
            "px;
  flex-direction: row;
  align-items: center;
  justify-content: center;
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ForwardRef>
          <ForwardRef>
            <Link
              replace={false}
              to="/"
            >
              <ForwardRef
                src="logo_small.png"
              />
            </Link>
            <ForwardRef>
              © Copyright 
              2019
              .
              <br />
              All right reserved.
            </ForwardRef>
            <ForwardRef>
              <ForwardRef>
                <FaFacebookF
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
              <ForwardRef>
                <FaTwitter
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
              <ForwardRef>
                <FaInstagram
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
            </ForwardRef>
          </ForwardRef>
        </ForwardRef>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <ForwardRef>
            <Link
              replace={false}
              to="/"
            >
              <ForwardRef
                src="logo_small.png"
              />
            </Link>
            <ForwardRef>
              © Copyright 
              2019
              .
              <br />
              All right reserved.
            </ForwardRef>
            <ForwardRef>
              <ForwardRef>
                <FaFacebookF
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
              <ForwardRef>
                <FaTwitter
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
              <ForwardRef>
                <FaInstagram
                  color="#808080"
                  size="30px"
                />
              </ForwardRef>
            </ForwardRef>
          </ForwardRef>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <Link
                replace={false}
                to="/"
              >
                <ForwardRef
                  src="logo_small.png"
                />
              </Link>,
              <ForwardRef>
                © Copyright 
                2019
                .
                <br />
                All right reserved.
              </ForwardRef>,
              <ForwardRef>
                <ForwardRef>
                  <FaFacebookF
                    color="#808080"
                    size="30px"
                  />
                </ForwardRef>
                <ForwardRef>
                  <FaTwitter
                    color="#808080"
                    size="30px"
                  />
                </ForwardRef>
                <ForwardRef>
                  <FaInstagram
                    color="#808080"
                    size="30px"
                  />
                </ForwardRef>
              </ForwardRef>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": <ForwardRef
                  src="logo_small.png"
                />,
                "replace": false,
                "to": "/",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "src": "logo_small.png",
                },
                "ref": null,
                "rendered": null,
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-EHOje",
                    "isStatic": true,
                    "rules": Array [
                      "
  height: 50px;
  width: 50px;
",
                    ],
                  },
                  "displayName": "styled.img",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-EHOje",
                  "target": "img",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  "© Copyright ",
                  2019,
                  ".",
                  <br />,
                  "All right reserved.",
                ],
              },
              "ref": null,
              "rendered": Array [
                "© Copyright ",
                2019,
                ".",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": "br",
                },
                "All right reserved.",
              ],
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-htpNat",
                  "isStatic": true,
                  "rules": Array [
                    "
  font-size: ",
                    "14",
                    "px;
  color: ",
                    "#808080",
                    ";
",
                  ],
                },
                "displayName": "styled.span",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-htpNat",
                "target": "span",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <ForwardRef>
                    <FaFacebookF
                      color="#808080"
                      size="30px"
                    />
                  </ForwardRef>,
                  <ForwardRef>
                    <FaTwitter
                      color="#808080"
                      size="30px"
                    />
                  </ForwardRef>,
                  <ForwardRef>
                    <FaInstagram
                      color="#808080"
                      size="30px"
                    />
                  </ForwardRef>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": <FaFacebookF
                      color="#808080"
                      size="30px"
                    />,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "color": "#808080",
                      "size": "30px",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": Object {
                    "$$typeof": Symbol(react.forward_ref),
                    "attrs": Array [],
                    "componentStyle": ComponentStyle {
                      "componentId": "sc-ifAKCX",
                      "isStatic": true,
                      "rules": Array [
                        "
  margin-left: ",
                        "22",
                        "px;
",
                      ],
                    },
                    "displayName": "styled.div",
                    "foldedComponentIds": Array [],
                    "render": [Function],
                    "styledComponentId": "sc-ifAKCX",
                    "target": "div",
                    "toString": [Function],
                    "warnTooManyClasses": [Function],
                    "withComponent": [Function],
                  },
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": <FaTwitter
                      color="#808080"
                      size="30px"
                    />,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "color": "#808080",
                      "size": "30px",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": Object {
                    "$$typeof": Symbol(react.forward_ref),
                    "attrs": Array [],
                    "componentStyle": ComponentStyle {
                      "componentId": "sc-ifAKCX",
                      "isStatic": true,
                      "rules": Array [
                        "
  margin-left: ",
                        "22",
                        "px;
",
                      ],
                    },
                    "displayName": "styled.div",
                    "foldedComponentIds": Array [],
                    "render": [Function],
                    "styledComponentId": "sc-ifAKCX",
                    "target": "div",
                    "toString": [Function],
                    "warnTooManyClasses": [Function],
                    "withComponent": [Function],
                  },
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": <FaInstagram
                      color="#808080"
                      size="30px"
                    />,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "color": "#808080",
                      "size": "30px",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": Object {
                    "$$typeof": Symbol(react.forward_ref),
                    "attrs": Array [],
                    "componentStyle": ComponentStyle {
                      "componentId": "sc-ifAKCX",
                      "isStatic": true,
                      "rules": Array [
                        "
  margin-left: ",
                        "22",
                        "px;
",
                      ],
                    },
                    "displayName": "styled.div",
                    "foldedComponentIds": Array [],
                    "render": [Function],
                    "styledComponentId": "sc-ifAKCX",
                    "target": "div",
                    "toString": [Function],
                    "warnTooManyClasses": [Function],
                    "withComponent": [Function],
                  },
                },
              ],
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-bxivhb",
                  "isStatic": true,
                  "rules": Array [
                    "
  display: flex;
  flex-direction: row;
",
                  ],
                },
                "displayName": "styled.div",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-bxivhb",
                "target": "div",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
          ],
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bwzfXH",
              "isStatic": true,
              "rules": Array [
                "
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;

  @media ",
                "(min-width: 320px)",
                " {  
    min-width: 100%;
  }

    @media ",
                "(min-width: 1024px)",
                " {  
    min-width: 1000px;
  }

  @media ",
                "(min-width: 2560px)",
                " {
    min-width: 1400px;
  }
",
              ],
            },
            "displayName": "styled.div",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bwzfXH",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": true,
            "rules": Array [
              "
  display: flex;
  margin: 0px;
  background-color: ",
              "#1f1f1f",
              ";
  height: 150px;
  padding: ",
              "14",
              "px;
  flex-direction: row;
  align-items: center;
  justify-content: center;
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
